src/MessageList/MessageListItem.vala

```diff
@@ -248,13 +257,26 @@ public class Mail.MessageListItem : Gtk.ListBoxRow {
 
         header_event_box.button_release_event.connect ((event) => {
             expanded = !expanded;
-            return false;
+            return Gdk.EVENT_STOP;
         });
 
         destroy.connect (() => {
             loading_cancellable.cancel ();
         });
 
+        starred_button.button_release_event.connect (() => {
```

Comment:
Why `button_release_event` instead of `clicked` ?

Reply:
So we can stop the event from propogating to the eventbox that collapses the mail

Reply:
That would be perfect to not change it in the future